.changesets/rename-gc_total_time-metric-to-gc_time.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "change"
+---
+
+Rename the (so far privately reported) `gc_total_time` metric to `gc_time`. It no longer reports the total time of Garbage Collection measured, but only the time between two (minutely) measurements.
```

Comment:
What is meant by "privately reported"?

Reply:
I mean we don't show it anywhere. We added it for the Ruby VM dashboard but then took the graph display it out at the last minute. People could make a graph themselves, but they'd have to find it in the reported metrics list and then they probably don't know what the values mean. So privately because it wasn't shown anywhere and it's not documented anywhere.